### ISSUE TITLE
Vault riddle does not reflect the problem at hand.

### ISSUE THREAD

**Issue by ghost** [d3athrow/vgstation13#21206]
![dreamseeker_xnniyhndk6](https://user-images.githubusercontent.com/20872793/50578769-d7d10000-0e03-11e9-8798-de541f98daf1.png)

The riddle provided on the paper for this vault states that you need to get all of the crates into the bottom right, but that does not reflect that you only need 12 to complete the puzzle.  The only way this can be seen is to meta the map by outside means.  The puzzle is lame, because the steps that are required to complete it would be early failure if going by the information provided.

This is the full picture with the amount of teleporters you need to fill.

![40987387-accc736e-68e8-11e8-9a0b-aafa7a668e5e](https://user-images.githubusercontent.com/20872793/50578822-8d9c4e80-0e04-11e9-8841-c94b180baf33.png)

This is very lame.

**Comment by MadmanMartian** [comment]
???

**Comment by ghost** [comment]
I needed to reword it to reflect the issue at hand.

**Comment by RealestEstate** [comment]
https://nethackwiki.com/wiki/Sokoban_Level_2a

**Comment by ghost** [comment]
@RealestEstate You are missing the point.  The point is that the paper hint in-game is not true to the riddle or the player's perception of it.  It leads the player to believe they will fail it if they miss a single crate, and there is no way to know the hole amounts without OOC knowledge.  This is lame.

**Comment by RealestEstate** [comment]
obviously there's more crates than holes so the implication is you don't use all

**Comment by Intigracy** [comment]
Just change the paper to reflect that

**Comment by RealestEstate** [comment]
@HarseTheef
no you're right, the paper should be fixed to reflect this fact.
